templates/AssetInventory/index.tsx

```diff
@@ -124,33 +124,36 @@ const AssetInventory = () => {
   };
 
   return (
-    <LoadingContainer loading={loading}>
-      <div className={classes.root}>
-        <HvControls
-          views={views}
-          callbacks={instance}
-          defaultView="card"
-          onViewChange={(evt, id) => setCurrentView(id)}
-        >
-          <HvLeftControl
-            placeholder="Search"
-            onSearch={handleSearch}
-            searchProps={{
-              inputProps: {
-                "aria-label": "Search",
-                "aria-controls": `${idsToControl.cards} ${idsToControl.list}`,
-              },
-            }}
-          />
-          <HvRightControl
-            values={rightControlValues}
-            sortProps={{
-              "aria-label": "Sort by",
+    <div className={classes.root}>
+      <HvControls
```

Comment:
I think we should disable the controls while it's loading to prevent the users from switching to the table view. Otherwise we get card skeletons when the view is a table.

Reply:
There's really no way to disable it without making changes to the `HvControls` component, which I don't think fall into the context of the work. I opted to hide the view switcher using the available prop on the component, which I think achieves the same goal.

Reply:
Makes sense to me 🙏